.circleci/config.yml

```diff
@@ -89,6 +89,9 @@ jobs:
       - run:
           name: install cloudify
           command: pip install -U cloudify==4.3.2
+      - run:
+          name: Install plugin wagon
+          command: wagon install workspace/build/cloudify_utilities_plugin*.wgn
       - run:
           name: install utilities plugin from branch
           command: pip install -e .
```

Comment:
Why do we need to install this in the local environment of the manager? We should have the wagon uploaded to the manager.